xarray/backends/rasterio_.py

```diff
@@ -322,11 +322,14 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None, loc
         attrs["units"] = riods.units
 
     # Parse extra metadata from tags, if supported
-    parsers = {"ENVI": _parse_envi}
+    parsers = {"ENVI": _parse_envi, "GTiff": lambda m: m}
 
     driver = riods.driver
     if driver in parsers:
-        meta = parsers[driver](riods.tags(ns=driver))
+        if driver == "GTiff":
```

Comment:
So the metadata that is read back depends on the driver used?

Reply:
yes, metadata can be driver dependent. there was already support for ENVI drivers and I followed this approach for geotiff. As it happens geotiff is a special case within GDAL and doesn't require specifying a namespace.